autosklearn/automl.py

```diff
@@ -1747,14 +1768,11 @@ def cv_results_(self):
 
         metric_mask = dict()
         metric_dict = dict()
-        metric_name = []
 
-        for metric in self._scoring_functions:
-            metric_name.append(metric.name)
+        for metric in itertools.chain(self._metrics, self._scoring_functions):
```

Comment:
Would it be possible, that two metrics have the same name, but do different things?

Reply:
Also, would it cause issues in l1823-l1840 if a metric/two metrics with the same name is/are in self._metrics **and** self._scoring_functions? And also l1847-l1858

Reply:
These are good suggestions for unit tests, I will add these in the next step.

Reply:
Okay, just finished them.